rxandroid/src/main/AndroidManifest.xml

```diff
@@ -12,4 +12,4 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<manifest package="rx.android" />
\ No newline at end of file
+<manifest package="io.reactivex.android" />
```

Comment:
This can be especially confusing if you'll mix RxJava (and/or RxAndroid) 1.x and 2.x in one project and especially in one file (I think a lot of devs will face that), not sure everybody will be able to immediately distinguish that `rx` means 1.x and `io.reactivex` means 2.x. cc @akarnokd